docker-compose.client.yml

```
@@ -7,3 +7,12 @@ services:
       - "${WORKSPACE:-./workspace}:/usr/src/app/workspace:ro"
       - "${REPORTS:-./workspace/reports}:/usr/src/app/reports:rw"
       - "${SUITES:-./suites}:/usr/src/app/suites:ro"
+  core:
+    build: ./core
+    network_mode: 'host'
```

```suggestion
    ports:
      - 80:80
```
No need for host network anymore.